can't build in VS2015, error

Hi，
 Today I build the leela-zero , I get these errors
1>------ 已启动生成: project: leela-zero, configuration: Release x64 ------
1>GameState.obj : error LNK2001: unresolved external symbol "public: __cdecl UCTNodePointer::~UCTNodePointer(void)" (??1UCTNodePointer@@QEAA@XZ)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: float __cdecl UCTNodePointer::get_eval(int)const " (?get_eval@UCTNodePointer@@QEBAMH@Z)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: bool __cdecl UCTNodePointer::active(void)const " (?active@UCTNodePointer@@QEBA_NXZ)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: float __cdecl UCTNodePointer::get_score(void)const " (?get_score@UCTNodePointer@@QEBAMXZ)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: int __cdecl UCTNodePointer::get_visits(void)const " (?get_visits@UCTNodePointer@@QEBAHXZ)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: bool __cdecl UCTNodePointer::valid(void)const " (?valid@UCTNodePointer@@QEBA_NXZ)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: void __cdecl UCTNodePointer::inflate(void)const " (?inflate@UCTNodePointer@@QEBAXXZ)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: class UCTNodePointer & __cdecl UCTNodePointer::operator=(class UCTNodePointer &&)" (??4UCTNodePointer@@QEAAAEAV0@$$QEAV0@@Z)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: __cdecl UCTNodePointer::UCTNodePointer(class UCTNodePointer &&)" (??0UCTNodePointer@@QEAA@$$QEAV0@@Z)
1>UCTNode.obj : error LNK2001: unresolved external symbol "public: __cdecl UCTNodePointer::UCTNodePointer(short,float)" (??0UCTNodePointer@@QEAA@FM@Z)
1>UCTNodeRoot.obj : error LNK2001: unresolved external symbol "public: int __cdecl UCTNodePointer::get_move(void)const " (?get_move@UCTNodePointer@@QEBAHXZ)

------------------------------------------------------------------------
Are some packages miss？

--- a/msvc/VS2015/leela-zero.vcxproj
+++ b/msvc/VS2015/leela-zero.vcxproj
@@ -126,7 +126,6 @@
     <ClInclude Include="..\..\src\UCTSearch.h" />
     <ClInclude Include="..\..\src\Utils.h" />
     <ClInclude Include="..\..\src\Zobrist.h" />
-    <ClInclude Include="Header.h" />
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />

--- a/msvc/VS2015/leela-zero.vcxproj
+++ b/msvc/VS2015/leela-zero.vcxproj
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
     <ProjectConfiguration Include="Debug|x64">
@@ -93,6 +93,7 @@
     <ClCompile Include="..\..\src\Training.cpp" />
     <ClCompile Include="..\..\src\Tuner.cpp" />
     <ClCompile Include="..\..\src\UCTNode.cpp" />
+    <ClCompile Include="..\..\src\UCTNodePointer.cpp" />
     <ClCompile Include="..\..\src\UCTNodeRoot.cpp" />
     <ClCompile Include="..\..\src\UCTSearch.cpp" />
     <ClCompile Include="..\..\src\Utils.cpp" />
@@ -121,9 +122,11 @@
     <ClInclude Include="..\..\src\Training.h" />
     <ClInclude Include="..\..\src\Tuner.h" />
     <ClInclude Include="..\..\src\UCTNode.h" />
+    <ClInclude Include="..\..\src\UCTNodePointer.h" />
     <ClInclude Include="..\..\src\UCTSearch.h" />
     <ClInclude Include="..\..\src\Utils.h" />
     <ClInclude Include="..\..\src\Zobrist.h" />
+    <ClInclude Include="Header.h" />
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />

